README.md

```diff
@@ -90,3 +90,7 @@ Please read [CONTRIBUTING.md](CONTRIBUTING.md) to get started with making contri
 ## Code of Conduct
 `TBD`
 
+## Steps to create docker image
+docker build --build-arg VERSION=0.1.4 -t yourapp .
```

Comment:
This might be better in the Getting Started section of the README. What do you think @sunitakawane ?